tools/Elasticurl/Elasticurl.NET.csproj

```diff
@@ -5,6 +5,12 @@
     <TargetFramework>netcoreapp2.0</TargetFramework>

```

Comment:
should this match netcoreapp3.1 below?

Reply:
Well, I should probably add building Elasticurl to the CI (add it to the solution probably).

Reply:
Removed the 3.1 entry since it just breaks CI if we want to build Elasticurl too.